### PR TITLE
SignOut controller added and tested.

### PR DESCRIPTION
**Pull request by DeepakKumar03**
Added Signout Controller.
I have uncomment UserControllerTest.java file and was able to pass test cases for SignOut API endpoint.

### CODE REVIEW

File: quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java [hunk 1]
```diff
@@ -32,6 +35,9 @@
     @Autowired
     SignupBusinessService signupBusinessService;
 
+    @Autowired
+    SignOutBusinessService signOutBusinessService;
```
Comment (simpleprahalad) [inline quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java:39]: Where is signout business service file ?

**Review by simpleprahalad — CHANGES_REQUESTED** [review]
Please add the missing files.

File: quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java (hunk 1 above)
Comment (DeepakKumar03) [reply quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java:39]: I have added now.
Kindly check

File: quora-service/src/main/java/com/upgrad/quora/service/business/SignOutBusinessService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            return userAuthToken;
+
+    }
+
+    @Transactional(propagation = Propagation.REQUIRED)
+    public void updateAuthToken(final UserAuthTokenEntity userAuthToken){
+
+        UserEntity signedUser= userAuthToken.getUser();
+        userAuthToken.setLogoutAt(ZonedDateTime.now());
+        userAuthToken.setExpiresAt(ZonedDateTime.now());
```
Comment (simpleprahalad) [inline quora-service/src/main/java/com/upgrad/quora/service/business/SignOutBusinessService.java]: Why are we updating expiresAt ?The ticket talks only to update the logoutAt time. Please recheck

**Review by simpleprahalad — CHANGES_REQUESTED** [review]
Please check for my query.

File: quora-service/src/main/java/com/upgrad/quora/service/business/SignOutBusinessService.java (hunk 2 above)
Comment (DeepakKumar03) [reply quora-service/src/main/java/com/upgrad/quora/service/business/SignOutBusinessService.java]: I have made changes accordingly.
Kindly re check once again.

**Comment by ahans30** [comment]
We will merge this into the dev branch (for integrated testing purpose) and not the master branch. Please add a PR for merging this to dev instead.

**Pull request closed without merging**